README.md

```diff
@@ -23,6 +23,7 @@ This extension controller supports the following Kubernetes versions:
 
 | Version         | Support     | Conformance test results |
 | --------------- | ----------- | ------------------------ |
+| Kubernetes 1.20 | 1.20.0+     | N/A |
```

Comment:
Can you update the link in the PR release note:

```diff
- https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.20.md
+ https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md
```

Reply:
Thanks for catching this! 💕 Done